Resources/Prototypes/_Sunrise/Entities/Objects/Specific/NTUplink.yml

```
@@ -5,6 +5,7 @@
   name: Bluespace ERT visionary
   description: Ultra-innovative portable bluespace teleporter.
   suffix: Empty
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)